tests/phpunit/tests/ajax/CropImage.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+/**
+ * Admin Ajax functions to be tested.
+ */
+require_once ABSPATH . 'wp-admin/includes/ajax-actions.php';
+
+require_once ABSPATH . 'wp-admin/includes/class-wp-filesystem-base.php';
+require_once ABSPATH . 'wp-admin/includes/class-wp-filesystem-direct.php';
+
+/**
+ * Class for testing ajax crop image functionality.
+ *
+ * @group ajax
+ */
+class Tests_Ajax_CropImage extends WP_Ajax_UnitTestCase {
+
+	/**
+	 * @covers wp_ajax_crop_image
```

Comment:
Should probably note `@ticket 37750` here as well.

Reply:
Thank you! Fixed in 4a548660707894c1983fae48939ce1af07d98d78